src/Source/HashTiming.php

```diff
@@ -82,9 +82,9 @@ public function generate($size)
             $total -= $bytes;
             for ($i=1; $i < 3; $i++) {
                 $t1   = microtime(true);
-                $seed = mt_rand();
+                $seed = $_seed = uniqid(mt_rand() . lcg_value() . rand() . getmypid(), true);
```

Comment:
Could you name it `$initialSeed` or such? Had to read it a couple times to understand the double assignment otherwise (same below)